mavros/src/mavros/ftp.py

```diff
@@ -73,6 +76,7 @@ def open(self, path, mode):
         try:
             ret = open_(file_path=path, mode=m)
         except rospy.ServiceException as ex:
+            reset_server()
```

Comment:
I'm not sure that reset here is good idea. Higher level code should do that.

Reply:
Ok, I agree with you